GraphView/po/pl-local.po

```diff
@@ -0,0 +1,634 @@
+msgid "Add to bookmarks"
+msgstr "Dodaj do zakładek"
+
+#: GraphView/graphview.py:3859
+msgid "Quick View"
+msgstr "Szybki podgląd"
+
+#: GraphView/graphview.py:3903
+msgid "Web Connection"
+msgstr "Połączenie web"
+
+#: GraphView/graphview.py:3929
+msgid "Select tags for family"
+msgstr "Wybierz tagi dla rodziny"
+
+#: GraphView/graphview.py:3946
+msgid "Add father"
+msgstr "Dodaj ojca"
+
+#: GraphView/graphview.py:3955
+msgid "Add mother"
+msgstr "Dodaj matkę"
+
+#: GraphView/graphview.py:3972
+msgid "Children"
+msgstr "Dzieci"
+
+#: GraphView/graphview.py:3981
+msgid "Add child to family"
+msgstr "Dodaj dziecko do rodziny"
+
+#: GraphView/graphview.py:4052
+msgid "About Graph View"
+msgstr "O Graph View"
+
+#: GraphView/graphview.py:4242
+msgid "Adding Tags"
+msgstr "Dodawanie tagów"
+
+##: GraphView/graphview.py:4248
+#, python-format
+msgid "Adding Tags to person (%s)"
+msgstr "Dodawanie tagów do osoby (%s)"
+
+#: GraphView/graphview.py:4253
+#, python-format
+msgid "Adding Tags to family (%s)"
+msgstr "Dodawanie tagów do rodziny (%s)"
+
+#: GraphView/graphview.py:4336
+msgid "Add Child to Family"
+msgstr "Dodaj dziecko do rodziny"
+
+#: GraphView/graphview.py:4351
+#, python-format
+msgid "Delete %s?"
+msgstr "Usunąć %s?"
+
+#: GraphView/graphview.py:4352
+#, python-format
+msgid "Deleting the person [%s] will remove it from the database."
+msgstr "Usunięcie osoby [%s] usunie ją z bazy."
+
+#: GraphView/graphview.py:4364
+#, python-format
+msgid "Delete Person (%s)"
+msgstr "Usuń osobę (%s)"
+
+#: GraphView/graphview.py:4381
+#, python-format
+msgid "Delete family [%s]?"
+msgstr "Usunąć rodzinę [%s]?"
+
+#: GraphView/graphview.py:4382
+msgid "Deleting the family will remove it from the database."
+msgstr "Usunięcie rodziny spowoduje usunięcie jej z bazy."
+
+#: GraphView/graphview.py:4393
+#, python-format
+msgid "Delete Family [%s]"
+msgstr "Usuń Rodzinę [%s]"
+
+#: GraphView/graphview.py:4412
+#, python-format
+msgid "%s has been bookmarked"
+msgstr "%s został/a dodany/a do zakładek"
+
+#: GraphView/search_widget.py:73
+msgid "Persons from current graph"
+msgstr "Osoby z aktualnego wykresu"
+
+#: GraphView/search_widget.py:74
+msgid "Other persons from database"
+msgstr "Pozostałe osoby z bazy"
+
+#: GraphView/search_widget.py:226
+msgid "No persons found..."
+msgstr "Nie znaleziono osób..."
+
+#: GraphView/search_widget.py:382
+msgid ""
+"Search people in the current visible graph and database.\n"
+"Use <Ctrl+F> to make search entry active."
+msgstr ""
+"Szukaj osób w aktualnym widocznym wykresie i bazie.\n"
+"Użyj <Ctrl+F> aby aktywować pozycję wyszukiwania."
+
+#: GraphView/search_widget.py:384
+msgid "Search..."
+msgstr "Szukaj..."
+
+#: GraphView/search_widget.py:575
+#, python-format
+msgid "<b>%s:</b>"
+msgstr "<b>%s:</b>"
+
+#: GraphView/search_widget.py:661
+#, python-format
+msgid "Birth: %s"
+msgstr "Narodziny: %s"
+
+#: GraphView/search_widget.py:665
+#, python-format
+msgid "Death: %s"
+msgstr "Śmierć: %s"
```

Comment:
How it is translated in Gramps: "Śmierć" or "Zgon"?

Reply:
There is "Śmierć" in Gramps.